index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Tarikwa Portfolio</title>
+</head>
+
+<body>
+    <section class="main-section">
+        <section class="toolbar">
+            <ul class="nav">
+                <li class="item"><a href="#">My Logo</a></li>
+                <li class="item"><a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a></li>
+            </ul>
+        </section>
+        <section class="headline-section">
+            <h1>I'm Tarikwa <br> Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <P class="connect">LET'S CONNECT</P>
+            <a href="#"><img src="images/social_media_icons.png" alt=""></a>
+        </section>
```

Comment:
- [ ] It would be nice to use font-awesome for the social media icons and wrap them in a `ul` tag. You can then wrap each icon in the `li` tag instead of using the snapshot that you used in the project.

Reply:
I have changed the image into an icon but I couldn't find a similar one for Github and Medium icons.